src/carousel.ts

```diff
@@ -22,6 +26,34 @@ function carousel(id: string, curChild=0) {
         centerCard(curChild);
     });
 
+
+    //Enable Swipe on mobile devices only
+    hammer.set({enable: windowSize <= 768 ? true : false})
+    //Detect swipe left or right on divParent
+    hammer.on("swipeleft swiperight", function(e: any) {
+        const {type, isFinal} = e;
+
+        //Event listener fires mutiple times. Waits until events are done.
+        if(isFinal){
+            //Check if user swipes right (image moves right)
+            if(type === 'swiperight'){
+                if(curChild === 0){
```

Comment:
We are getting a lot of redundant code here.  Would be best to consolidate.  Perhaps replace the addEventListeners on divLeft and divRight with:

() => scrollBy(-1)  and () => scrollBy(1)

```
function scrollBy(n: number) {
  const nextChild = curChild + n;
  if (nextChild < 0 || nextChild > numChildren) {
     return;
  }
  curChild = nextChild;
  centerCard(curChild);
 showControls();
```
  
}